SharedPackages/AIChat/Tests/AIChatTests/AIChatPromptHandlerTests.swift

```diff
@@ -23,7 +23,7 @@ final class AIChatPromptHandlerTests: XCTestCase {
 
     func testSetData() {
         let handler = AIChatPromptHandler.shared
-        let testData = "Test Prompt"
+        let testData: AIChatPromptHandler.DataType = .init(platform: "platform", query: .init(prompt: "Test Prompt", autoSubmit: false))
```

Comment:
Not sure that this is the right change, feel free to offer a different suggestion 😅